sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/schema/BeamSqlRecordCoder.java

```diff
@@ -34,13 +35,12 @@
 import org.apache.beam.sdk.coders.InstantCoder;
 import org.apache.beam.sdk.coders.ListCoder;
 import org.apache.beam.sdk.coders.StringUtf8Coder;
-import org.apache.beam.sdk.extensions.sql.impl.utils.CalciteUtils;
 
 /**
- *  A {@link Coder} encodes {@link BeamSqlRow}.
+ *  A {@link Coder} encodes {@link BeamSqlRecord}.
  */
-public class BeamSqlRowCoder extends CustomCoder<BeamSqlRow> {
-  private BeamSqlRowType tableSchema;
+public class BeamSqlRecordCoder extends CustomCoder<BeamSqlRecord> {
```

Comment:
It would be preferable (and more useful) to add such coders to the SDK rather than hide the conversions here in BeamSqlRecordCoder. If you wanted to avoid exposing these coders you can make them inner classes of BeamSqlRecordTypeProvider (with a TODO to consider making them top-level).